modules/core/04-channel/v2/keeper/relay.go

```diff
@@ -146,27 +146,27 @@ func (k Keeper) recvPacket(
 		return channeltypes.ErrNoOpMsg
 	}
 
-	path := hostv2.PacketCommitmentKey(packet.SourceId, packet.Sequence)
+	path := hostv2.PacketCommitmentKey(packet.SourceChannel, packet.Sequence)
 	merklePath := types.BuildMerklePath(counterparty.MerklePathPrefix, path)
 
 	commitment := channeltypesv2.CommitPacket(packet)
 
 	if err := k.ClientKeeper.VerifyMembership(
 		ctx,
-		packet.DestinationId,
+		packet.DestinationChannel,
```

Comment:
Likewise, lookup v2 channelEnd and use the clientID rather than channelID

Reply:
nice catch!! Though it seems this was already wrong before

Reply:
Yup!